File: Python/PGS/LEVEL2/124 나라의 숫자/gunwoo.py
def solution(n):
    answer = ""
    while n > 0:
        n -= 1
        an = n%3
        n = n//3
        
        if an == 0:
            answer = "1" + answer
        elif an == 1:
            answer = "2" + answer
        elif an == 2:
            answer = "4" + answer
                      
    return answer

File: Python/PGS/LEVEL1/제일 작은 수 제거하기/gunwoo.java
class Solution {
    public int[] solution(int[] arr) {
        if (arr.length == 1) {
            return new int[]{-1};
        }
        
        int[] answer = new int[arr.length-1];
        int small = arr[0];
        int index = 0;
        for (int i = 1; i < arr.length; i++) {
            if (arr[i] < small) {
                small = arr[i];
            }
        }
        for (int i = 0; i < arr.length; i++){
            if (arr[i] != small){
                answer[index] = arr[i];
                index++;
            }
        }
        return answer;
    }
}